src/tracer.c

```diff
@@ -2019,7 +2026,11 @@ void __tracer_unverified_snap_to_active(struct snap_device *dev,
                 goto error;
 
         // generate the full pathname
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5,5,0)
```

Comment:
if I get the codebase corectly we try not to use kernel versions numbers directly, but rather go with feature flags

Reply:
It cannot be solved using script generated ifdefs becasue functions are hidden from using (you cannot include proper headers)